.github/workflows/ci.yml

```diff
@@ -20,14 +20,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: ['3.7', '3.8', '3.9', '3.10']
+        python: ['3.10', '3.11', '3.12']
         django: ['3.2', '4.1', '4.2']
```

Comment:
django 3.2 supports py310 as maximum version, so you'll need to exclude 311 and 312 from the build matrix.

I don't think that any Django version supports python 3.12 since it was only released two weeks ago, I suspect 4.2 will gain support and 5.0, but older versions seem unlikely. Confirmed by https://code.djangoproject.com/ticket/34118